sample-apps/springboot/build.gradle.kts

```diff
@@ -24,6 +24,8 @@ dependencies {
   implementation("com.squareup.okhttp3:okhttp")
   implementation("software.amazon.awssdk:s3")
   implementation("io.opentelemetry:opentelemetry-api")
+  implementation("io.opentelemetry.instrumentation:opentelemetry-oshi:0.11.0")
```

Comment:
Can you put on alphabetical order?

Reply:
Ah also can you move the versions to dependencyManagement project like our others? Especially instrumentation version will drift easily